contracts/CharityChallenge.sol

```diff
@@ -14,7 +14,12 @@ contract CharityChallenge {
 
     address payable public contractOwner;
 
+    // wanna keep this as first npo address for compatibility
```

Comment:
because we haven't deployed on mainnet yet, it's okay to remove this. Front-end change should be in to accommodate this